backend/package.json

```
@@ -33,6 +33,7 @@
     "@dotenvx/dotenvx": "1.38.3",
     "@elastic/elasticsearch": "7.17.14",
     "@frontegg/client": "5.3.2",
+    "@napi-rs/canvas": "^0.1.80",
```

lock the versions here
```suggestion
    "@napi-rs/canvas": "0.1.80",
```